include/af/signal.h

```diff
@@ -753,13 +753,23 @@ extern "C" {
 /**
    C Interface for signals interpolation on one dimensional signals.
 
-   \param[out] out the interpolated array.
-   \param[in]  in is the multidimensional input array. Values assumed to lie uniformly spaced indices in the range of `[0, n)`, where `n` is the number of elements in the array.
-   \param[in]  pos positions of the interpolation points along the first dimension.
-   \param[in]  method is the interpolation method to be used. The following types (defined in enum \ref af_interp_type) are supported: nearest neighbor, linear, and cubic.
-   \param[in]  off_grid is the default value for any indices outside the valid range of indices.
-   \return     \ref AF_SUCCESS if the interpolation operation is successful,
-               otherwise an appropriate error code is returned.
+   \param[in,out] out the interpolated array.
+   \param[in]     in is the multidimensional input array. Values assumed to lie
+                  uniformly spaced indices in the range of `[0, n)`, where `n`
+                  is the number of elements in the array.
+   \param[in]     pos positions of the interpolation points along the first
```

Comment:
Could you make the variables more prominent? They are currently merged into the description.
Example:
```
\param[in]     pos positions of the interpolation points along the first
                   dimension.
```